CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project 1.4.x per each release will be documented in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [v1.4.0.5] - TBA
```

Comment:
@kaumini can we please set the release date here and commit it to the development `1.4.x` branch via a separate PR?

Reply:
Fixed